docs/tutorials/light-up.md

```diff
@@ -0,0 +1,224 @@
+---
+title: "Use Object Detection to Turn Your Lights On"
+linkTitle: "Turn on Lights with Object Detection"
+weight: 49
+type: "docs"
+tags: ["camera", "vision", "detector", "python"]
+description: "How to turn a light on when your webcam sees a person."
+# Author: Hazal Mestci
+---
+
+In this tutorial, we will show you how to build your very own object detection robot using Viam, your computer with a webcam, a [Kasa Smart Plug](https://www.kasasmart.com/us/products/smart-plugs), and a desk lamp.
+This robot will turn the lights on or off when it detects a person in front of it.
+You can turn it into a night light for reading books, a security robot that alerts you when a person is close by, or a bathroom light that only activates when people enter; the opportunities are endless.
+
+This project is a great place to start if you are new to building robots, and have a smart plug laying around.
+
+{{< gif webm_src="../img/light-up/light-up.webm" mp4_src="../img/light-up/light-up.mp4" alt= "The project working: a person sitting at a desk with a computer and lightbulb set up in front of her. As she leaves the light turns off, and as she enters the frame, the light turns back on." >}}
+
+## Hardware requirements
+
+The following hardware is used in this tutorial:
+
+- Computer with a webcam
+  - This tutorial uses a MacBook Pro but any computer running macOS or 64-bit Linux will work
+- Mobile phone (to download the Kasa Smart app)
+- [Kasa Smart Wi-Fi Plug Mini](https://www.kasasmart.com/us/products/smart-plugs/kasa-smart-wifi-plug-mini)
+- [Kasa Smart Light Bulb](https://www.kasasmart.com/us/products/smart-lighting/kasa-smart-light-bulb-multicolor-kl125)
+- [Table Lamp Base](https://www.amazon.com/gp/product/B08KZNZVY7/ref=ppx_yo_dt_b_asin_title_o02_s00?) or similar
+
+## Software requirements
+
+The following software is used in this tutorial:
+
+- [Python3](https://www.python.org/downloads/)
+- [`viam-server`](/installation/install/)
+- [Viam Python SDK](https://python.viam.dev/)
+- [Project repo on GitHub](https://github.com/viam-labs/devrel-demos/tree/main/Light%20up%20bot)
+
+## Prerequisites
+
+Make sure you have a recent version of Python installed on your computer by running
+
+```bash
+python --version
+```
+
+If the output is Python 3.9.0 or later, you should be fine for this tutorial.

```

Comment:
Is this the version this was tested on or do we actually need some functionality that was only added in Python 3.9? I would assume anything 3.0+ is probably fine. 

I think you can remove everything until line 48 and then merge the installation instructions for the SDK with the list above.